package.json

```diff
@@ -26,19 +26,24 @@
     "url": "https://github.com/mapbox/geojson-mapnikify/issues"
   },
   "homepage": "https://github.com/mapbox/geojson-mapnikify",
+  "files": [
+    "index.js",
+    "bin",
+    "lib"
+  ],
   "dependencies": {
-    "@kartotherian/blend": "^2.0.1",
+    "@mapbox/blend": "^2.0.1",
     "@mapbox/geojson-normalize": "~0.0.1",
-    "@wikimedia/makizushi": "^4.0.0",
-    "agentkeepalive": "^4.0.2",
-    "minimist": "~1.2.0",
+    "@wikimedia/makizushi": "4.1.0",
```

Comment:
Seems fine to automatically pull in minor updates?

Reply:
Done.